api/datadoghq/common/datadogpodautoscaler_types.go

```diff
@@ -199,6 +199,11 @@ type DatadogPodAutoscalerConstraints struct {
 	// MaxReplicas is the upper limit for the number of POD replicas. Needs to be >= minReplicas.
 	MaxReplicas int32 `json:"maxReplicas"`
 
+	// MaxUnreadyPodsPercent is the maximum percentage of unready pods allowed before pausing scaling. Disabled by default.
```

Comment:
Would it immediately resume scaling when the percentage of unready pods drops below the threshold? Maybe it's worth mentioning in the description.